src/modules/displayMovies.js

```diff
@@ -0,0 +1,27 @@
+const cardsContainer = document.querySelector('.cards-display-section');
+
+const displayMovies = async (shows) => {
+  cardsContainer.innerHTML = '';
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < 20 && i < shows.length; i++) {
```

Comment:
You should not disable eslint, as it is there for help the developer to write industry standard code. Requesting your attention to  line no 5 of "displayMovies.js"

Reply:
Hi @Nahid1911 ,
Thanks for the constructive feedback, I have made the change✔